### ISSUE TITLE
We have too many stopwords in Solr

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#640]
Got a complaint today that searches for "during the term of" brought back weird results. Sure enough, "during", "the" and "of" are all on our current stop words list. 

Since we're about to reindex everything, we should revisit the stopwords list, purge it of most of its entries and land it before the reindex occurs. 

Part of the reason for stopwords is performance, but, frankly, I've heard a lot of people saying *any* stopwords list is unnecessary, and on top of that, the default list in Solr is empty. 

Our list has something like 200 entries, causing issues as above. Let's trim it.

**Comment by voutilad** [comment]
Curious...is this currently being done at index time or query time? I can't recall which. I think if there's going to be a big reindexing some of the index-time stuff might be best to turn off and move to the query parser if possible.

**Comment by mlissner** [comment]
It's on both index and query. It's on Index because the whole point is keeping your index from getting huge. It's on query because if it weren't you'd get no results whenever you searched for a stopword. It's annoying to have to reindex whenever your stopword or synonym file changes, but I don't think there's a way around it.

**Comment by voutilad** [comment]
What's the total index doc count these days? I know people used to optimize heavily for small indices, but these days it seems overkill unless it's tuning for a particular corpus.

It does looks like the CL list is larger than some English defaults I'm finding online.

- [CourtListener](https://github.com/freelawproject/courtlistener/blob/master/Solr/conf/lang/stopwords_en.txt): 177 entries
- [Solr's Example in Master](https://git-wip-us.apache.org/repos/asf?p=lucene-solr.git;a=blob;f=solr/example/files/conf/lang/stopwords_en.txt;h=2c164c0b2a1e57254b60ad2c0b1d113bad743911;hb=refs/heads/master#l22): 33 entries

**Comment by mlissner** [comment]
Different indexes:

 - Opinions: ~4M bigish documents, will go to ~6M when we finish our next import.
 - RECAP: About to be ~20M documents. 1M are full docs, others are just summaries (i.e. small)

Glad to see they put something in master. I looked in Solr 4.10 and it was totally empty. 

I just trimmed our original list down and then added any that were missing from Solr Master. Seems like a solid improvement. Rolling with it, thanks.
